packages/wdio-sumologic-reporter/package.json

```diff
@@ -33,5 +33,9 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "devDependencies": {
+    "@types/dateformat": "^3.0.1",
+    "@types/json-stringify-safe": "^5.0.0"
```

Comment:
I think we need to move them into the dependency section otherwise we run into problems when compiling the project for a release (here we don't install dev dependencies).